### PR TITLE
[TACHYON-1031] Move IndexedSet to Common

### PR DESCRIPTION
**Pull request by aaudiber**

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

File: servers/src/test/java/tachyon/master/IndexedSetTest.java [hunk 1]
```diff
@@ -24,6 +24,8 @@
 import org.junit.Before;
 import org.junit.Test;
 
+import tachyon.IndexedSet;
+
 public class IndexedSetTest {
```
Comment (gpang) [inline servers/src/test/java/tachyon/master/IndexedSetTest.java]: Should the unit test also move to common/ ?

**Comment by gpang** [comment]
I left a minor question. Thanks for the fix!

**Comment by yupeng9** [comment]
LGTM

File: servers/src/test/java/tachyon/master/IndexedSetTest.java (hunk 1 above)
Comment (aaudiber) [inline servers/src/test/java/tachyon/master/IndexedSetTest.java]: Good catch, I'm now moving the test as well.

**Comment by gpang** [comment]
LGTM

**Pull request merged**